4-css-grid/styles.css

```diff
@@ -0,0 +1,18 @@
+* section {
+    background-color: rgb(66, 66, 66);
+    border-radius: 15px;
+    color: whitesmoke;
+    margin: 5px;
+    padding: 20px;
+}
+body {
+    display: grid;
+    grid-template: 50% 50% / 1fr 1fr 1fr;
+}
+section:first-child {
```

Comment:
cool, you got the child pseudo selectors, but prefer classnames so your markup doesn't have to match your styles.